full/src/test/java/apoc/redis/RedisTest.java

```diff
@@ -0,0 +1,375 @@
+
+        TestUtil.testCall(db, "CALL apoc.redis.scard($uri, $key, $config)",
+                map("uri", URI, "key", key, "config", config),
+                r -> assertEquals(3L, r.get("value")));
+
+        TestUtil.testCall(db, "CALL apoc.redis.smembers($uri, $key, $config)",
+                map("uri", URI, "key", key, "config", config),
+                r -> assertEquals(Set.of("foo", "bar", "baz"), new HashSet<>(fromCodecList((List<Object>) r.get("value")))));
+
+        TestUtil.testCall(db, "CALL apoc.redis.spop($uri, $key, $config)",
+                map("uri", URI, "key", key, "config", config),
+                r -> assertTrue(List.of("foo", "bar", "baz").contains(fromCodec(r.get("value")))));
+
+        TestUtil.testCall(db, "CALL apoc.redis.scard($uri, $key, $config)",
+                map("uri", URI, "key", key, "config", config),
+                r -> assertEquals(2L, r.get("value")));
+    }
+
+    @Test
+    public void testSortedSetsCommands() {
+        Map<String, Object> config = map("codec", this.codec);
+        Object key = getByCodec("mySortedSetKey");
+        TestUtil.testCall(db, "CALL apoc.redis.zadd($uri, $key, [0, $first, 100, $third, 1, $second], $config)",
+                map("uri", URI, 
+                        "first", getByCodec("first"), "second", getByCodec("second"), "third", getByCodec("third"),
+                        "key", key, "config", config),
+                r -> assertEquals(3L, r.get("value")));
+        
+        TestUtil.testCall(db, "CALL apoc.redis.zcard($uri, $key, $config)",
+                map("uri", URI, "key", key, "config", config),
+                r -> assertEquals(3L, r.get("value")));
+        
+        TestUtil.testCall(db, "CALL apoc.redis.zrangebyscore($uri, $key, 0, 100, $config)",
+                map("uri", URI, "key", key, "config", config),
+                r -> assertEquals(List.of("first", "second", "third"), fromCodecList((List<Object>) r.get("value"))));
+        
+        TestUtil.testCall(db, "CALL apoc.redis.zrem($uri, $key, $members, $config)",
+                map("uri", URI, "members", getListByCodec(List.of("first", "second")), "key", key, "config", config),
+                r -> assertEquals(2L, r.get("value")));
+
+        TestUtil.testCall(db, "CALL apoc.redis.zrangebyscore($uri, $key, 0, 100, $config)",
+                map("uri", URI, "key", key, "config", config),
+                r -> assertEquals(List.of("third"), fromCodecList((List<Object>) r.get("value"))));
+    }
+
+    @Test
+    public void testHashesCommands() {
+        Map<String, Object> config = map("codec", this.codec);
+        Object key = getByCodec("mapKey");
+        Object numberValue = getByCodec("number");
+        TestUtil.testCall(db, "CALL apoc.redis.hset($uri, $key, $field, $value, $config)",
+                map("uri", URI, "key", key, "field", numberValue, "value", getByCodec("1"), "config", config),
+                r -> assertEquals(true, r.get("value")));
```

Comment:
Yes, the `HSET` command returns this, but the corresponding Java command returns a boolean (true if the field is new, otherwise false).
To be clearer, I updated the procedure description with this note.
https://lettuce.io/lettuce-4/release/api/com/lambdaworks/redis/api/sync/RedisHashCommands.html#hset-K-K-V-